packages/TimePicker/tests/TimePicker.specs.js

```diff
@@ -0,0 +1,153 @@
+import TimeInterpreter from "../src/TimeInterpreter";
```

Comment:
does this spec fill actually run? i noticed file name is `specs.js` but we usually just have `spec.js`